src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirDataStore.cs

```diff
@@ -440,11 +413,64 @@ private static string RemoveTrailingZerosFromMillisecondsForAGivenDate(DateTimeO
             return formattedDate.Replace(milliseconds, trimmedMilliseconds, StringComparison.Ordinal);
         }
 
-        private static string RemoveVersionIdAndLastUpdatedFromMeta(ResourceWrapper resourceWrapper)
+        private void ReplaceVersionIdAndLastUpdatedInMeta(ResourceWrapper resourceWrapper)
+        {
+            var date = GetJsonValue(resourceWrapper.RawResource.Data, "lastUpdated");
+            string rawResourceData;
+            if (resourceWrapper.Version == "1") // version is already correct
```

Comment:
Low priority: I see there are plenty of places where there are checks validating if the version of a resource is "1". 

Usually, we try to avoid "Magic Numbers". I would suggest creating a constant in the code called "InitialVersion" or "FirstVersion" with the value "1". Then the code will be more readable in that sense.

Reply:
Will do